Fabric.Identity.API/Models/ModelExtensions.cs

```diff
@@ -58,8 +58,10 @@ public static IS4.Client ToIs4ClientModel(this Client client)
                     PostLogoutRedirectUris = client.PostLogoutRedirectUris,
                     EnableLocalLogin = client.EnableLocalLogin,
                     IdentityProviderRestrictions = client.IdentityProviderRestrictions,
-                    LogoutUri = client.LogoutUri,
-                    LogoutSessionRequired = client.LogoutSessionRequired,
+                    FrontChannelLogoutUri = client.LogoutUri,
+                    BackChannelLogoutUri = client.LogoutUri,

```

Comment:
We specify that the AllowedGrantTypes for a Client is Implicit, which is FrontChannel. Why we using BackChannel?